fe/fe-core/src/main/java/org/apache/doris/common/util/VectorizedUtil.java

```diff
@@ -37,7 +37,10 @@ public static boolean isVectorized() {
         if (stmtExecutor == null) {
             return connectContext.getSessionVariable().enableVectorizedEngine();
         }
-        Analyzer analyzer = stmtExecutor.getAnalyzer();
+        Analyzer analyzer = null;
```

Comment:
Your pr should not bring this change, is it caused by no rebase master?

Reply:
revert it~